tests/sweep_framework/sweeps/matmul/short/matmul_user_program_config_mcast_1d.py

```diff
@@ -103,7 +103,7 @@ class TensorMemoryConfigs(enum.Enum):
                 memory_layout=ttnn.TensorMemoryLayout.WIDTH_SHARDED,
                 buffer_type=ttnn.BufferType.L1,
                 shard_spec=ttnn.ShardSpec(
-                    ttnn.experimental.tensor.num_cores_to_core_range_set(35, core_grid, row_wise=True),
+                    ttnn._ttnn.tensor.num_cores_to_core_range_set(35, core_grid, row_wise=True),
                     (64, IN0_INNER_DIM_PER_CORE),
```

Comment:
I'm assuming you will change this to ttnn.num_cores_to_core_range_set

Reply:
yes, it will be changed in @VirdhatchaniKN 's work split PR